test/test_databases/test_postgres.py

```diff
@@ -0,0 +1,179 @@
+from parsons import Postgres
+from parsons.etl.table import Table
+from test.utils import assert_matching_tables
+import unittest
+import os
+import re
+import warnings
+import datetime
+from test.utils import validate_list
+
+# The name of the schema and will be temporarily created for the tests
+TEMP_SCHEMA = 'parsons_test'
+
+# These tests do not interact with the Postgres Database directly, and don't need real credentials
+
+
+class TestPostgres(unittest.TestCase):
```

Comment:
I would suggest renaming it for more clarity. `TestPostgresCreateTable`

Reply:
Fixed.